Lib/unittest/mock.py

```diff
@@ -1760,6 +1761,7 @@ def method(self, *args, **kw):
     '__hash__': lambda self: object.__hash__(self),
     '__str__': lambda self: object.__str__(self),
     '__sizeof__': lambda self: object.__sizeof__(self),
+    '__fspath__': lambda self: "%s/%s" % (type(self).__name__, id(self))
```

Comment:
missing trailing comma.

Add a comma at the end of this line to ease reviews that add items in the future.